### ISSUE TITLE
no reply when run interact_mmi.py

### ISSUE THREAD

**Issue by GaloisGroGauss** [yangjianxin1/GPT2-chitchat#61]
There is no reply from chatbot when running  interact_mmi.py. 
Meanwhile, the file mmi_samples.txt is created with empty contents.
In the case of interact.py, no issues occur.
environment: Python3.7, Pytorch1.3.1, transformers2.1.1, in MacOS Big Sur

**Comment by GaloisGroGauss** [comment]
like this:
2021-01-21 20:05:06,291 - INFO - using device:cpu
开始和chatbot聊天，输入CTRL + Z以退出
user:你是谁？
chatbot:
user:不理我
chatbot:
user:哼
chatbot:
user:怎么回事
chatbot:
user:^Z
zsh: suspended  python3 interact_mmi.py --no_cuda

**Comment by GaloisGroGauss** [comment]
断点调试了一下代码，发现是 interact_mmi.py的以下几行代码造成的问题。
            mmi_input_tensor = torch.tensor(mmi_input_id).long().to(device)
            out = mmi_model(input_ids=mmi_input_tensor, labels=mmi_input_tensor)
            # loss为nan的时候就会出问题
            loss = out[0].item()
加上后面的
             if loss < min_loss:
                    best_response = response
                    min_loss = loss
就是如果mmi模型计算的第一个机器回答的loss为nan的话，那就惨了。
所以，请问作者，这个mmi_model什么时候会搞出loss为nan以及如何修正？

**Comment by GaloisGroGauss** [comment]
烦请作者看看 @yangjianxin1
